src/wp-includes/block-bindings/pattern-overrides.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Pattern Overrides source for the Block Bindings.
+ *
+ * @since 6.5.0
+ * @package WordPress
+ * @subpackage Block Bindings
+ */
+
+/**
+ * Gets value for the Pattern Overrides source.
+ *
+ * @since 6.5.0
+ * @access private
+ *
+ * @param array    $source_args    Array containing source arguments used to look up the override value.
+ *                                 Example: array( "key" => "foo" ).
+ * @param WP_Block $block_instance The block instance.
+ * @param string   $attribute_name The name of the target attribute.
+ * @return mixed The value computed for the source.
+ */
+function _block_bindings_pattern_overrides_get_value( array $source_args, $block_instance, string $attribute_name ) {
```

Comment:
This kind of private function is in general better as inline functions/closures. I think these are not authorized in Core yet right? Do you know more about the reasoning...

Reply:
I was thinking about it, too. I’ll ask around and check the codebase. The only benefit of using another function is the PHPDoc included.

Reply:
You definitely can find places where inline functions are defined as params passed. You definitely shouldn't use them when passing the callback to WP hooks, but here, it would make perfect sense. I'll confirm that and refactor the code, if possible, after adding more unit tests.